aws/data_source_aws_vpc_peering_connections.go

```diff
@@ -46,9 +44,6 @@ func dataSourceAwsVpcPeeringConnectionsRead(d *schema.ResourceData, meta interfa
 	if err != nil {
 		return err
 	}
-	if resp == nil || len(resp.VpcPeeringConnections) == 0 {
-		return fmt.Errorf("no matching VPC peering connections found")
-	}
```

Comment:
To prevent potential panics, we should still check for `nil`:

```suggestion
	if resp == nil
		return fmt.Errorf("error reading EC2 VPC Peering Connections: empty response")
	}
```

Reply:
Thanks! Done.